.github/workflows/bump-dependencies.yml

```diff
@@ -0,0 +1,36 @@
+name: Bump dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 8 * * 1' # At 08:00 on Monday.
+
+jobs:
+  bump-dependencies:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+
+      - run: python -m pip install poetry pre-commit
+      - run: poetry lock
+      - run: poetry export --without-hashes --dev -f requirements.txt -o requirements_dev.txt -E python
+      - run: poetry export -f requirements.txt --without-hashes -o requirements.txt
+      - run: pre-commit autoupdate
+      - run: git status
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          branch: bump-dependencies
+          delete-branch: true
+          commit-message: Bump dependencies
+          title: 'Bump dependencies'
+          body: |
+            Weekly update of backend dependencies.
+          labels: |
```

Comment:
Wouldn't `team-reviewers` be useful? It would allow to ping whenever the PR is open

Reply:
I usually manage it myself and if not, the other backend leads do. For now, I think it's not necessary.